src/Resources/contao/modules/ModuleArticle.php

```diff
@@ -366,7 +366,7 @@ public function generatePdf()
 		// Close and output PDF document
 		$pdf->lastPage();
 		$pdf->Output(standardize(ampersand($this->title, false)) . '.pdf', 'D');
-
+		// FIXME: This has to be changed to a ResponseException somehow - rather write to temp file and stream it then?
```

Comment:
I said " I don't have a valid use case for StreamedResponse in Contao, yet.". I did not say I don't see a valid use case for streaming at all. I just said the `BinaryFileResponse` is the best fit for us as it does stream as well ;)